ports/donottakethiscathome/Do Not Take This Cat Home.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+XDG_DATA_HOME=${XDG_DATA_HOME:-$HOME/.local/share}
+
+if [ -d "/opt/system/Tools/PortMaster/" ]; then
+  controlfolder="/opt/system/Tools/PortMaster"
+elif [ -d "/opt/tools/PortMaster/" ]; then
+  controlfolder="/opt/tools/PortMaster"
+elif [ -d "$XDG_DATA_HOME/PortMaster/" ]; then
+  controlfolder="$XDG_DATA_HOME/PortMaster"
+else
+  controlfolder="/roms/ports/PortMaster"
+fi
+
+source $controlfolder/control.txt
+[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
+get_controls
+
+# Variables
+PORTEXEC="renpy/startRENPY"
+GAMEDIR="/$directory/ports/donottakethiscathome"
+
+cd $GAMEDIR
+
+runtime="renpy_8.3.4"
+if [ ! -f "$controlfolder/libs/${runtime}.squashfs" ]; then
+  # Check for runtime if not downloaded via PM
+  if [ ! -f "$controlfolder/harbourmaster" ]; then
+    pm_message "This port requires the latest PortMaster to run, please go to https://portmaster.games/ for more info."
+    sleep 5
+    exit 1
+  fi
+
+  $ESUDO $controlfolder/harbourmaster --quiet --no-check runtime_check "${runtime}.squashfs"
+fi
+
+# Savedata setup
+mkdir -p $GAMEDIR/conf
+export XDG_DATA_HOME="$GAMEDIR/conf"
+bind_directories ~/.renpy/ $GAMEDIR/conf/.portfolder 
+
+renpydir="$GAMEDIR/renpy/"
```

Comment:
Not a good idea to double up on these slashes.

```suggestion
renpydir="$GAMEDIR/renpy"
```